src/main/java/org/amalitech/ContactBook.java

```diff
@@ -0,0 +1,54 @@
+package org.amalitech;
+
+import java.util.ArrayList;
+
+public class ContactBook {
+ private String name;
+ private String email;
+ ArrayList<Contact> contacts;
+
+ public ContactBook(String name, String email){
+     this.email=email;
```

Comment:
Why does ContactBook take a name and an email? 
IMO I think the contact book is just keeping track of a list of Contacts hence has no need for a name and an email.